tests/Tests/400__ods_dashboard/adminUITest.robot

```diff
@@ -0,0 +1,159 @@
+
+    Check User Management Option Is Available For The User     ${OCP_ADMIN_USER.USERNAME}   ${OCP_ADMIN_USER.PASSWORD}  ${OCP_ADMIN_USER.AUTH_TYPE}
+    Remove OpenShift Groups To Data Science Administrators     rhods-admins
+    Save Changes In User Management Setting
+    Switch Browser  1
+    Add OpenShift Groups To Data Science Administrators         rhods-noaccess
+    Add OpenShift Users Groups To Data Science Administrators   rhods-users
+
+    Verify Expect Changes In AdminGroups Are Present In CRD     cluster-admins
+    Verify Expect Changes In AllowedGroups Are Present In CRD   system:authenticated
+    Save Changes In User Management Setting
+    Page Should Contain  Unable to load User and group settings
+    Switch Browser  2
+    [Teardown]  Teardown For Admin UI   ${OCP_ADMIN_USER.USERNAME}   ${OCP_ADMIN_USER.PASSWORD}  ${OCP_ADMIN_USER.AUTH_TYPE}
+
+Verify User Is Able To Spawn Jupyter Notebook
+    [Documentation]     Verify Correct User Is able to Spawn Jupyter Notebook
+    [Tags]   ODS-1681
+    Set Library Search Order  SeleniumLibrary
+    Check User Management Option Is Available For The User   ${TEST_USER.USERNAME}   ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
+    Clear All The Input Values In User Managermant
+    Add OpenShift Groups To Data Science Administrators         rhods-admins
+    Add OpenShift Users Groups To Data Science Administrators   rhods-admins
+    Save Changes In User Management Setting
+    Launch JupyterHub Spawner From Dashboard     ${TEST_USER.USERNAME}   ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
+    [Teardown]  Teardown For Admin UI  ${TEST_USER.USERNAME}   ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
+
+Verify User Is Not Able To Spawn Jupyter Notebook
+    [Documentation]    Verify User Is Not Able To Spawn Jupyter Notebook
+    [Tags]   ODS-1680
+    Set Library Search Order  SeleniumLibrary
+    Check User Management Option Is Available For The User   ${TEST_USER.USERNAME}   ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
+    Clear All The Input Values In User Managermant
+    Add OpenShift groups To Data Science Administrators         rhods-admin
+    Add OpenShift Users groups To Data Science Administrators   rhods-admin
+    Save Changes In User Management Setting
+    Close Browser
+    Launch Dashboard  ocp_user_name=${TEST_USER_3.USERNAME}  ocp_user_pw=${TEST_USER_3.PASSWORD}  ocp_user_auth_type=${TEST_USER_3.AUTH_TYPE}
+    ...               dashboard_url=${ODH_DASHBOARD_URL}  browser=${BROWSER.NAME}  browser_options=${BROWSER.OPTIONS}
+    Menu.Navigate To Page    Applications    Enabled
+    Wait Until Page Contains    Launch application
+    Click Element               //*[@class="odh-card__footer__link"]
+    Wait Until Page Contains    Page Not Found   timeout=15
+    Page Should Contain         Page Not Found
+    [Teardown]  Teardown For Admin UI   ${TEST_USER.USERNAME}   ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
```

Comment:
the same checks are implemented in ODS-293. A pr has been already raised to adapt the TC to use CRD instead of config map. I think it's not necessary to repeat the check in this suite

Reply:
we can remove it ODS-1681
but i would like to keep ODS-1680 as its there will be changes in CRD In future

Reply:
ok. Which future changes in CRD are you referring to?